js/script.js

```diff
@@ -0,0 +1,35 @@
+var prefix = "https://cors-anywhere.herokuapp.com/",
+	tweetLink = "https://twitter.com/intent/tweet?text=",
+	quoteUrl = "https://quotesondesign.com/wp-json/posts?filter[orderby]=rand&filter[posts_per_page]=1";
+	
+function getQuote() {
+	$.getJSON(quoteUrl, createTweet);
+	$.getJSON(prefix + quoteUrl, createTweet);
+	$.ajaxSetup({ cache: false });
+}
+	
+function createTweet(input) {
+	var data = input[0];
```

Comment:
input is your response, before you use this response data, check that input is really an array with at least one element